drivers/net/bnxt/bnxt_hwrm.c

```diff
@@ -950,16 +964,22 @@ int bnxt_hwrm_vnic_qcfg(struct bnxt *bp, struct bnxt_vnic_info *vnic,
 	HWRM_CHECK_RESULT;
 
 	vnic->dflt_ring_grp = rte_le_to_cpu_16(resp->dflt_ring_grp);
-	vnic->fw_rss_cos_lb_ctx = rte_le_to_cpu_16(resp->rss_rule);
-	// vnic->cos_rule = rte_le_to_cpu_16(resp->cos_rule);
-	// vnic->lb_rule = rte_le_to_cpu_16(resp->lb_rule);
+	vnic->rss_rule = rte_le_to_cpu_16(resp->rss_rule);
+	vnic->cos_rule = rte_le_to_cpu_16(resp->cos_rule);
+	vnic->lb_rule = rte_le_to_cpu_16(resp->lb_rule);
```

Comment:
This matches the API definition.  While only one rule is allowed at a time by present firmware, the API allows all three to be independently configured.